integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -492,15 +548,18 @@ public void testConfigClusterStartServerUsingAlways() {
    * by patching the resource definition with 
    *  spec/managedServers/2/serverStartPolicy set to ALWAYS.
    * Make sure that managed server managed-server2 is up and running
+   * Verify that if server start policy is ALWAYS and the server is selected
```

Comment:
See the comment for testConfigClusterStartServerUsingAlways(), the same is applicable to this method

Reply:
same reason as above